src/main/java/com/aws/greengrass/util/platforms/windows/WindowsPlatform.java

```diff
@@ -84,9 +86,13 @@
     public Set<Integer> killProcessAndChildren(Process process, boolean force, Set<Integer> additionalPids,
                                                UserDecorator decorator)
             throws IOException, InterruptedException {
+        if (!force) {
+            // TODO support graceful kill
+            throw new UnsupportedOperationException();
+        }
+
         PidProcess pp = Processes.newPidProcess(process);
         ((WindowsProcess) pp).setIncludeChildren(true);
-        ((WindowsProcess) pp).setGracefulDestroyEnabled(true);
```

Comment:
you don't need to remove this and throw unsupported. This does work correctly, but only for GUI apps.